src/overlays/actors/ovl_En_Kanban/z_en_kanban.c

```diff
@@ -879,7 +879,7 @@ void EnKanban_Draw(Actor* thisx, GlobalContext* globalCtx) {
             }
             gDPSetPrimColor(POLY_XLU_DISP++, 0x00, 0x00, 0, 0, 0, (s8)shadowAlpha);
 
-            if ((this->actionState == ENKANBAN_SIGN) && (gSaveContext.linkAge == 1)) {
+            if ((this->actionState == ENKANBAN_SIGN) && (LINK_IS_NOT_ADULT)) {
```

Comment:
```suggestion
            if ((this->actionState == ENKANBAN_SIGN) && LINK_IS_NOT_ADULT) {
```
occurs in a few spots, simple find&replace in all files (be careful about `if (LINK_IS_NOT_ADULT)` though)
may want to check other macros for useless parens as well, I also saw at least `(CHECK_QUEST_ITEM(QUEST_ZORA_SAPPHIRE))`